### DISCUSSION TITLE
Guest Book

### DISCUSSION THREAD

**Discussion by LuftigerLuca** [LuftigerLuca/LuftigerLuca#2] [General]
Hey this is my guest book, you can leave a nice message here

**Comment by 0skxr** [comment]
🅱️👍

**Comment by Riftra** [comment]
Hey

**Comment by maxmielchen** [comment]
Hey, I'm with the current maintainer of **UntisAPI** and saw that you did a fork with a useful feature, if you want you can make a pull request so we can adopt this feature. If not, I will implement this feature myself.
